src/sentry/hybridcloud/models/webhookpayload.py

```diff
@@ -67,8 +79,17 @@ class Meta:
             ),
         )
 
+        constraints = [
+            models.CheckConstraint(
+                check=Q(destination_type=DestinationType.SENTRY_REGION)
+                & Q(region_name__isnull=False),
+                name="webhookpayload_region_name_not_null",
+            ),
+        ]
```

Comment:
this introduces a new warning:

/Users/asottile/workspace/sentry/src/sentry/hybridcloud/models/webhookpayload.py:83: RemovedInDjango60Warning: CheckConstraint.check is deprecated in favor of `.condition`.
  models.CheckConstraint(

surprised this didn't get caught in CI but please fix 🙏